recipe/meta.yaml

```diff
@@ -17,11 +17,11 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.6,<3.10
     - pip
     - pytest-runner
   run:
-    - python >=3.6
+    - python >=3.6,<3.10
```

Comment:
Due to build errors [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=430459&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=1424), @AndrewSonwat and I found this to be the most reasonable way forward to unblock this release. The underlying problem seems to be coming from trafaret being incompatible with Python 3.10 (due to the [backwards incompatible changes they made to the collections module](https://docs.python.org/3.9/library/collections.html)). I've created a ticket to ensure trafaret gets updated by v2.27.1.